astroquery/vizier/core.py

```diff
@@ -205,44 +205,44 @@ def find_catalogs(self, keywords, *, include_obsolete=False, verbose=False,
         Examples
         --------
         >>> from astroquery.vizier import Vizier
-        >>> catalog_list = Vizier.find_catalogs('Kang W51') # doctest: +REMOTE_DATA +IGNORE_WARNINGS
-        >>> catalog_list # doctest: +REMOTE_DATA +IGNORE_OUTPUT
-        OrderedDict([('J/ApJ/684/1143', </>), ('J/ApJ/736/87', </>) ... ])
-        >>> print({k:v.description for k,v in catalog_list.items()}) # doctest: +REMOTE_DATA +IGNORE_OUTPUT
-        {'J/ApJ/684/1143': 'BHB candidates in the Milky Way (Xue+, 2008)',
-         'J/ApJ/736/87': 'Abundances in G-type stars with exoplanets (Kang+, 2011)',
-         'J/ApJ/738/79': "SDSS-DR8 BHB stars in the Milky Way's halo (Xue+, 2011)",
-         'J/ApJ/760/12': 'LIGO/Virgo gravitational-wave (GW) bursts with GRBs (Abadie+, 2012)',
-         ...}
+        >>> catalog_list = Vizier.find_catalogs('Mars') # doctest: +REMOTE_DATA +IGNORE_WARNINGS
+        >>> for k, v in catalog_list.items(): # doctest: +REMOTE_DATA +ELLIPSIS
```

Comment:
Is the order of return deterministic? If not, I don't think we can drop the IGNORE_OUTPUT. (Also, I believe ELLPSIS is always on, or at least it should be)

Reply:
It should be deterministic, but new ones can be added. I'll add the ignore output again just in case